integration/tests/cluster.sh

```diff
@@ -92,8 +92,13 @@ do_test_deploy() {
     export ELASTICSEARCH_URL="http://$frontend1_ip:10144"
     test_notifications_endpoint="http://$test_container_ip:15555"
 
-    run_inspec_tests "${A2_ROOT_DIR}" "a2-iam-v2-integration"
+    # Inspec tests are less tolerant of transient 500s; we expect a few of
+    # those as bad postgres connections get purged from the various services.
+    # Restart everything before running the tests
+    docker exec -t "$_frontend1_container_name"  "$cli_bin" restart-services
```

Comment:
i'm not sure i understand whats happening here. We've just done a deploy. Why does restarting everything help

Reply:
The frontends come up with broken Postgres connections, so the Inspec tests get exactly 2 500s, see for example https://buildkite.com/chef/chef-automate-master-verify-private/builds/9454#d349a780-69be-4a0f-ac17-10f6ae8f2475. Retrying 500s in the Inspec would be the better solution but it's a lot harder.

Reply:
For anyone who might stumble on this later, the code comment above is the correct answer--an HAProxy somewhere has the idle timeout set to 5 minutes, which kills the connections on frontend1 during the more-than-five-minutes when frontend2 is being deployed.